app/.gitlab-ci.yml

```diff
@@ -266,9 +266,9 @@ test:backend-format:
     default: false
   script:
     - cd app/backend
-    - autoflake --exclude src/proto -r -i --remove-all-unused-imports --check src
-    - isort --check --diff .
-    - black --check --diff .
+    - ruff check .
+    - ruff check --diff .
+    - ruff format --diff .
```

Comment:
Is there a way to combine these three into one command?

Reply:
It does not seem to be available yet: https://github.com/astral-sh/ruff/issues/8232

I feel this is fine, although `ruff check .` and `ruff check --diff .` will have overlap. The first will show all lint rules being broken while the second will show the proposed diffs for all lint violations that can be fixed automatically. But let me know if you have a differing opinion on this.

Reply:
Yes that sounds good. It would be good if it showed all the errors then fails the build if there are any.